static/app/views/issueDetails/groupReplays/useReplaysFromIssue.tsx

```diff
@@ -70,10 +54,10 @@ function useReplayFromIssue({
       version: 2,
       fields: REPLAY_LIST_FIELDS,
       projects: [Number(group.project.id)],
-      query: `id:[${String(response.replayIds)}]`,
+      query: `id:[${String(replayIds)}]`,
```

Comment:
Should this have an explicit stats period that's shared with `/replay-count/` request?

Reply:
ya, lets get that in